common/model/editorial-series.ts

```diff
@@ -1,4 +1,9 @@
-export type EditorialSeriesColour = 'turquoise' | 'red' | 'green' | 'purple';
+export type EditorialSeriesColour =
+  | 'teal'
```

Comment:
Could this rely on the definition of ColorSelection so we don't have to remember to keep them both in sync?

Reply:
It definitely could be defined in 1 place, I'll work with Gareth to see what these values should be.